airflow/api_fastapi/core_api/serializers/variables.py

```diff
@@ -31,7 +31,7 @@ class VariableBase(BaseModel):
     model_config = ConfigDict(populate_by_name=True)
 
     key: str
-    description: str | None
+    description: str | None = Field(default=None)
```

Comment:
@pierrejeambrun, I assign these as `Field` and set their `default` to `None` for backward compatibility, because these fields are nullable for `POST` and `PATCH` operations. It is a bit repetitive.

Reply:
Yes we need to do that for all our `Body` models, to respect the `legacy API` contract. (if fields could be missing, It should be possible in the new API to omit them)

Reply:
Yeap, I have included these to Body models.